main.py

```diff
@@ -0,0 +1,116 @@
+from Board import Board
+from Snake import Snake
+from Ladder import Ladder
+
+import random
+
+def roll_die():
+    return random.randint(1, 6)
+
+
+def create_board():
```

Comment:
This method should be the part of Board class.

Reply:
user input should not be taken in Board class. seems fine to take it here